src/scales/quantitativeScale.ts

```diff
@@ -256,6 +257,24 @@ export class QuantitativeScale<D> extends Scale<D, number> {
     }
   }
 
+  public magnify(magnifyAmount: number, centerValue: number) {
```

Comment:
Not all `Plottable.Scale`s are transformable (e.g. color scales). So, I added the `TransformableScale` interface to the particular scales that implement it. Quantitative scales implements this interface and are essentially abstract. Plottable doesn't use abstract types, but instead implements error-throwing methods, so i followed that pattern.